charts/cosigned/templates/webhook/sa_webhook.yaml

```diff
@@ -6,3 +6,7 @@ metadata:
     control-plane: {{ template "cosigned.fullname" . }}-webhook
   name: {{ template "cosigned.fullname" . }}-webhook
   namespace: {{ .Release.Namespace }}
+  {{- with .Values.serviceAccount.annotations }}
```

Comment:
this does not exist in the values.yaml file

the one I can see is `webhook.service. annotations`

Reply:
I bumped the chart version and add the block to the values.yaml file, thanks!